snapcraft/plugins/v2/cmake.py

```diff
@@ -69,6 +69,6 @@ def _get_cmake_configure_command(self) -> str:
     def get_build_commands(self) -> List[str]:
         return [
             self._get_cmake_configure_command(),
-            'cmake --build . -- -j"${SNAPCRAFT_PARALLEL_BUILD_COUNT}"',
-            'cmake --build . --target install -- DESTDIR="${SNAPCRAFT_PART_INSTALL}"',
+            'cmake --build . -j"${SNAPCRAFT_PARALLEL_BUILD_COUNT}"',
+            'cmake --install . --prefix="${SNAPCRAFT_PART_INSTALL}"',
```

Comment:
This doesn't look equivalent to what you're replacing: `--prefix` is described as setting `CMAKE_INSTALL_PREFIX`, which is different to `DESTDIR`.

Reply:
I've already removed it from the last revision.